### ISSUE TITLE
version 9 ready

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-android#84]

**Comment by tomholub** [comment]
@DenBond7 do you have an estimated date for v9 to be ready to test?

**Comment by DenBond7** [comment]
Yes, I do. I think it will be a next week. I need to complete few tasks and
test the app using different email providers.

Now I already added support different email providers. But while I testing
the app I found different things which work different than in Gmail. I need
to fix it.

**Comment by tomholub** [comment]
Got it, looking forward to test it

**Comment by DenBond7** [comment]
Maybe It will be more fast, but I'm not sure.

**Comment by tomholub** [comment]
Testing it, so far looks good on my device

**Comment by tomholub** [comment]
Currently, I cannot successfully test the following:

**download manager** https://github.com/FlowCrypt/flowcrypt-android/issues/82
 - waiting for #107

**searching private key backup** https://github.com/FlowCrypt/flowcrypt-android/issues/105 
- one user still had issues.. I have no clue why. I'll have to investigate.

**acra reports** https://github.com/FlowCrypt/flowcrypt-android/issues/6
 - a debug build crashed on me, but no report showed on backend. Phone showed "App crashed" and "restart"

**Comment by DenBond7** [comment]
> waiting for #107

completed

> acra reports #6

completed

**Comment by tomholub** [comment]
Great, I'll do tests over the weekend
